test/teos/unit/test_internal_api.py

```diff
@@ -45,6 +50,17 @@ def internal_api(db_manager, gatekeeper, carrier, block_processor):
     i_api.rpc_server.stop(None)
 
 
+@pytest.fixture()
+def clear_state(internal_api, db_manager):
```

Comment:
I'm guessing setting the db fixture to be cleared every test did not work?

Reply:
I didn't try that, but I thought that it would be better to explicitly mark tests that rely on empty db.